site2/docs/admin-api-overview.md

```diff
@@ -1,39 +1,34 @@
 ---
 id: admin-api-overview
-title: The Pulsar admin interface
+title: Pulsar admin interface
 sidebar_label: Overview
 ---
 
-The Pulsar admin interface enables you to manage all of the important entities in a Pulsar [instance](reference-terminology.md#instance), such as [tenants](reference-terminology.md#tenant), [topics](reference-terminology.md#topic), and [namespaces](reference-terminology.md#namespace).
+The Pulsar admin interface enables you to manage all important entities in a Pulsar instance, such as tenants, topics, and namespaces.
 
-You can currently interact with the admin interface via:
+You can interact with the admin interface via:
 
-- Making HTTP calls against the admin {@inject: rest:REST:/} API provided by Pulsar [brokers](reference-terminology.md#broker). For some restful apis, they might be redirected to topic owner brokers for serving
-   with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you are using `curl`, you should specify `-L`
-   to handle redirections.
-- The `pulsar-admin` CLI tool, which is available in the `bin` folder of your [Pulsar installation](getting-started-standalone.md):
-
-```shell
-$ bin/pulsar-admin
-```
+- Making HTTP calls against the admin {@inject: rest:REST:/} API provided by Pulsar brokers. For some restful APIs, they might be redirected to the topic owner brokers for serving with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you use `curl`, you should specify `-L` to handle redirections.
```

Comment:
```suggestion
- Making HTTP calls against the admin {@inject: rest:REST:/} API provided by Pulsar brokers. For some RESTful APIs, they might be redirected to the owner brokers for serving with [`307 Temporary Redirect`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307), hence the HTTP callers should handle `307 Temporary Redirect`. If you use `curl` commands, you should specify `-L` to handle redirections.
```

I think these three bullets  should use the same structure. you can try to update this as:
- HTTP calls, which is made against ...
- Java client interfacce
- The`pulsar-admin` CLI tool, which